app/models/blog.rb

```diff
@@ -1,5 +1,13 @@
 class Blog < ApplicationRecord
-  has_many :comment
   belongs_to :user
-  has_many :like
+  has_many :comments
+  has_many :likes
+
```

Comment:
- [ ] **OPTIONAL**
Please consider using the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) `after_save` in order to call the `update_post` method every time a new post has been saved, then make the method private